Figgle.Generator.Tests/RenderTextSourceGeneratorTests.cs

```diff
@@ -18,6 +18,7 @@ public void SimpleCase_Class()
     {
         string source =
             """
+            using Figgle;
```

Comment:
NOTE: `GenerateFiggleText` is defined under the `Figgle` namespace and thus when using the attribute we must include `using Figgle;`.  The original tests didn't include `using Figgle` but was able to work because the old generator worked off the attribute's name.

Now that we're using `IIncrementalGenerator`, it requires the fully qualified name of the `GenerateFiggleText` attribute matches `Figgle.GenerateFiggleText`, so we must make sure the unit test is including the `Figgle` namespace correctly.